pyprep/find_noisy_channels.py

```diff
@@ -539,7 +599,10 @@ def run_ransac(
 
         # Memory seems to be fine ...
         # Make the predictions
-        n_chns, n_timepts = data.shape
+        # n_chns, n_timepts = data.shape
+        # 2 next lines should be equivalent but support single channel processing
+        n_timepts = data.shape[1]
+        n_chns = chn_pos.shape[0]
```

Comment:
why is `n_chns, n_timepts = data.shape` not working anymore for single chn proc?

Reply:
Because n_chns corresponds to the channels being predicted, not all the set of channels. Before since we did all the predictions at once the numbers were the same but now we only predict one channel at a time. 

Now, the number of channels predicted is defined through the chn_pos we want to interpolate. Currently this will always yield 1 since is only 1 channel.

I decided to keep it general because I was thinking a more optimized way of doing this was to predict in bash of m channels that doesn't go above the ram supplied. But that may be in the future.

Reply:
Actually I ended up doing the optimization. I think the solution is quite elegant. Looking forward to your suggestions.